### PR TITLE
Basic changes to the readme and form ui

### PR DESCRIPTION
**Pull request by mpmckenna8**
Tried to fill in the README.md with some simple instructions for people who might want to get the app running to use it locally or contribute front end code who may have very little golang background.  Also tried to edit the form-template.html to make it easier for users to enable editing mode.

### CODE REVIEW

File: stuff/template/form-template.html [hunk 1]
```diff
@@ -119,27 +119,54 @@
...
+          elements[i].style["background-color"] = '#eeeeee';
+          tx.textContent = "(off)"
+
       }
-  }
+        }
+        form_is_enabled = enable_action;
+        edit_toggle = document.getElementsByClassName("edit-toggle");
+        if (edit_toggle.length > 0) {
+          for(i of edit_toggle){
```
Comment (hzeller) [inline stuff/template/form-template.html]: Interesting, is this a way to iterate through containers in JavaScript ? If this is something that only works in newer browsers still consider just iterating through an array.
(But please add a space between `for` and `(`)

File: stuff/template/form-template.html [hunk 2]
```diff
@@ -119,27 +119,54 @@
...
-	   ? '/static/edit-pen.png'
-	   : '/static/non-edit-pen.png';
+
+    var form_is_enabled;
+
+    function enable_form(enable_action) {
+        var elements = document.querySelectorAll("#compform input:not(.nav-item),textarea");
+        for (var i = 0; i < elements.length; ++i) {
+          elements[i].disabled = !enable_action;
+          var tx = document.getElementById("edittext")
```
Comment (hzeller) [inline stuff/template/form-template.html]: Why do you do this in the inside of the loop ? The edittext element won't change.

File: stuff/template/form-template.html [hunk 3]
```diff
@@ -119,27 +119,54 @@
...
+
+
+      if (enable_action) {
+          elements[i].style["background-color"] = "white";
+          tx.textContent = "(on)"
```
Comment (hzeller) [inline stuff/template/form-template.html]: Same here: for every input element you set the global text-field.
You can do that simply once at the beginning of the function; something like
```
   document.getElementById("edittext").textContent = enable_action ? "(on)" : "(off)"
```

File: README.md [hunk 4]
```diff
@@ -11,8 +11,22 @@ Uses SQLite to keep data in one file, so you need this external go dependency
...
+To run the app navigate to the /stuff directory and run:
+
+```
+go run main.go dbbackend.go search-handler.go search.go form-handler.go image-handler.go resistor-image.go template-renderer.go status-handler.go sitemap-handler.go
+```
+
+Or you can navigate to the /stuff directory and run:
+
+```
+make all
```
Comment (mpmckenna8) [inline README.md]: Cool.  I figured out the long go run command just by trial and error trying to get it to run never having made a go thing run before.  But those are good tips and I'll change it just use make stuff.  I should probably note they should be working in a Bash or similar shell for this to work?

File: stuff/template/form-template.html (hunk 2 above)
Comment (mpmckenna8) [inline stuff/template/form-template.html]: no good reason, I was still figuring out what was going on... moved out.

File: stuff/template/form-template.html [hunk 5]
```diff
@@ -232,20 +274,23 @@
...
+            {{end}}
+      </td>
 	  </tr>
 
 	  <tr>
 	    <td align="right"><label for="fprint">Footprint</label></td>
-	    <td><input type="text" name="footprint" size="10" id="fprint" value="{{.Footprint}}">
+	    <td>
+        <input type="text" name="footprint" size="10" id="fprint"
+         value="{{.Footprint}}">
```
Comment (mpmckenna8) [inline stuff/template/form-template.html]: Sounds good. but I was finding it really hard to follow the multiple element super long lines so is it ok to keep those line breaks?

**Comment by mpmckenna8** [comment]
So I did just leave in the two icons, because I think they were extra confusing b/c the cursor didn't change when you hovered over it before.  Now it looks like a link to a user and with both things changing it shouldn't be too confusing.  But maybe removing the bottom one is a good idea.
If you want to change the text I had feel free to too.  

Also I moved all the JS from a bunch of script tags to one. Hope that's okay as I found it weird to have a script tag in the head.

**Comment by hzeller** [comment]
Have a look, I have updated things in the last submit including your suggestion with the pointer (but only one button) https://github.com/hzeller/stuff-org/commit/c0c8d469186fa2026cac1b5df36e6a1048a82e55
